js/interface-advanced-directory.js

```diff
@@ -652,7 +652,7 @@ var DataDirectoryForm = (function() {
         listviewTemplate: _this.listviewEditor.getValue(),
         detailviewTemplate: _this.detailviewEditor.getValue(),
         customCss: _this.customCssEditor.getValue(),
-        customJs: _this.customJsEditor.getValue()
+        customJs: 'Fliplet().then(function() { ' + _this.customJsEditor.getValue() + ' })'
```

Comment:
I was thinking more about adding it to the html, otherwise

1. it's visible to the end user
2. you're adding it every time you save :)

Reply:
True true.